### PR TITLE
Starting a presentation in Google Slides

### LINKED ISSUE #1440: "Start presentation" Google Slides
Similar to #1115 a user should be able to start the presentation of a Google Slide deck by saying things like.

* Start slideshow
* Present
* Start presentation
* Presenter view

### PR DESCRIPTION
**Pull request by dave-ok**
Before submitting a final PR, please:

- [x] Run `npm test` on your machine
- [x] Run `npm run format` to keep code formatted consistently
- [x] Reference the issue you are fixing in at least one of your commit messages, as `Fixes #X`

Fixes #1440 

@ianb I have created this intent and placed it the slideshow intent folder. I don't know if this is okay. It can be easily moved to a different location if this is unsuitable. Please when you get the chance could you review it? Thanks

### CODE REVIEW

File: extension/intents/slideshow/presentationScript.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/* globals communicate */
+
+this.presentationScript = (function() {
+  const PRESENT_BUTTON_SELECTOR = "#punch-start-presentation-left";
+  const SLIDE_THUMBNAILS = ".punch-filmstrip-thumbnail";
+
+  function clickElement(element) {
+    element.dispatchEvent(new MouseEvent("mousedown"));
+    element.dispatchEvent(new MouseEvent("mouseup"));
```
[the remark this answers is not in the document]
Comment (ianb) [reply extension/intents/slideshow/presentationScript.js:9]: Well, I thought I'd give it a try, and no, `.click()` doesn't work, so...

[the remark this answers is not in the document]
Comment (dave-ok) [reply extension/intents/slideshow/presentationScript.js:9]: Yes I discovered this after much fiddling. There was no click event listener

File: extension/intents/slideshow/presentationScript.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  function startPresentation() {
+    if (document.querySelectorAll(SLIDE_THUMBNAILS).length === 0) {
```
[the remark this answers is not in the document]
Comment (dave-ok) [reply extension/intents/slideshow/presentationScript.js:13]: Sorry. Leftover habits from another life. :)

**Comment by dave-ok** [comment]
@ianb Thank you very much. I hope its okay to make PRs now

**Pull request merged**